libs/prov/prov_k8s_cortx_deploy.py

```diff
@@ -280,7 +280,10 @@ def deploy_cluster(self, node_obj: LogicalNode, remote_code_path: str) -> tuple:
         return : True/False and resp
         """
         LOGGER.info("Deploy Cortx cloud")
-        cmd = common_cmd.DEPLOY_CLUSTER_CMD.format(remote_code_path, self.deploy_cfg['log_file'])
+        export_cmd = common_cmd.LINUX_EXPORT.format(self.deploy_cfg["deploy_ha_timeout_key"],
```

Comment:
Better to use `os.environ` than `export` command

Reply:
export we have used it for setting the environment value on remote host.